launcher-env-template.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/sh
+
+#################################################
+# Here you can find basic setup for this script #
+#################################################
+
+# Choose (uncomment) one of the 3 KeyCloak options below
```

Comment:
I was thinking that we could perhaps put an `if` around this to test if this value is already set in the environment. If so we just use that one. That makes it possible to switch without having to edit this file, if that's what you prefer. Wdyt?